core/lib.deno_core.d.ts

```diff
@@ -38,5 +38,8 @@ declare namespace Deno {
 
     /** Get heap stats for current isolate/worker */
     function heapStats(): Record<string, number>;
+
+    /** Encode a string to its Uint8Array representation. */
+    function encode(input: string): Uint8Array;
```

Comment:
Why does this need to be a Deno.core API? Can't it be a normal op?

Reply:
It is already a Deno.core API (`Deno.core.encode`). This just adds it to our internal typings. It has no effect on runtime.

Reply:
But to answer your question, yeah can be a normal op. Let's tackle that in a followup though, this PR is large enough as is.